tests/e2e/basic-frontend/02-assert.yaml

```diff
@@ -33,6 +33,4 @@ spec:
               protocol: TCP
           volumeMounts:
             - name: config
-              mountPath: /opt/app-root/src/build/stable/operator-generated
-            - name: config
-              mountPath: /opt/app-root/src/build/preview/operator-generated
+              mountPath: /opt/app-root/src/build/stable/operator-generated
```

Comment:
new line?

Reply:
no need, file is properly formatted. I'd love to pass through lint and automatic format processes to be honest.